pkgs/applications/window-managers/i3/pystatus.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, libpulseaudio, python3Packages, extraLibs ? [] }:
+
+python3Packages.buildPythonApplication rec {
+  name = "${pname}-${version}";
+  version = "3.34";
+  pname = "i3pystatus";
+  disabled = !python3Packages.isPy3k;
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/i/${pname}/${name}.tar.gz";
+    sha256 = "1bpkkf9q4zqq7fh65zynbv26nq24rfznmw71jjvda7g8kjrwjdk5";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ keyring colour netifaces praw psutil basiciw ] ++
+    [ libpulseaudio ] ++ extraLibs;
+
+  ldWrapperSuffix = "--suffix LD_LIBRARY_PATH : \"${libpulseaudio}/lib\"";
+  makeWrapperArgs = [ ldWrapperSuffix ]; # libpulseaudio.so is loaded manually
+
+  postInstall = ''
+    makeWrapper ${python3Packages.python.interpreter} $out/bin/${pname}-python-interpreter \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      ${ldWrapperSuffix}
+  '';
```

Comment:
@domenkozar, configuration file for the i3pystatus is just a python script that can be run within `i3bar` like this:
```
#i3bar
bar {
    status_command    python ~/.path/to/your/i3pystatus/configuration/file.py
    position          top
    workspace_buttons yes
}
```
In my own i3-configuration I use it in this way:
```
bar {
    status_command i3pystatus-python-interpreter ~/.i3/status-config.py
    position top
    workspace_buttons yes
}
```
where `i3pystatus-python-interpreter` is generated by `postInstall` phase with all necessary libs.
I could use `i3pystatus` application for my purposes, but it is an example application that produces clock time and nothing else. It can't load configuration file like `xmobar` or `i3status`.
I didn't find better solution.